pytorch_toolkit/object_detection/README.md

```diff
@@ -1,23 +1,25 @@
-## Object Detection
+# Object detection
 
+## Pre-trained models
 
-### Face Detection
+This repo contains scripts and tutorials for object detection models training.
 
-This repository contains training scripts for the lightweight SSD-based face detector. The detector is based on the MobileNetV2 backbone and has a single SSD head with manually designed anchors. As a result, it has computational complexity 0.51 GMACs and 1.03 M of parameters.
+### [Face Detection](face_detection.md)
 
-### Person Vehicle Bike Detection
+Models that are able to detect faces on given images.
 
-The detection network model provides detection of 3 class objects: vehicle, pedestrian, non-vehicle (ex: bikes).
-This detector was trained on the data from crossroad cameras.
+### [Person Vehicle Bike Detection](person_vehicle_bike_detection.md)
+
+Models that are able to detect 3 classes of objects: person, vehicle, non-vehicle (e.g. bike).
 
 ## Setup
 
 ### Prerequisites
 
 * Ubuntu\* 16.04
-* Python\* 3.6
-* PyTorch\* 1.0.1
-* OpenVINO™ 2019 R1 with Python API
```

Comment:
Maybe it'd make sense to put a link to `external/mmdetection` in the "Installation" section for its build instructions and to avoid confusions with the upstream `mmdetection`.

Reply:
added  a link to external/mmdetection to Prerequisites
init_venv contains a path to correct mmdetection already

Reply:
Cool. BTW, have you tried running `init_venv.sh`?

Reply:
good joke

Reply:
fixed for older version of git (ubuntu 16.04)